### ISSUE TITLE
About training loss

### ISSUE THREAD

**Issue by L3LeTrigger-F** [RUCAIBox/LC-Rec#8]
Hi,we are training RQ-VAE with the code in this link: https://github.com/RUCAIBox/LC-Rec/tree/main/index
Why did total loss go down first and then up, but reconstruction loss went down gradually?
We ran experiments on our own data set, this is the trend of loss:

![8d3760ba-9d29-42f9-8626-3eb0d8fdeeea_capture](https://github.com/RUCAIBox/LC-Rec/assets/43927725/a9b8b8d0-af3f-4ef7-af24-8634a2a64680)

We also repeated the experiment on the original data set and found that the change trend of loss is the same. So what is the reason?

**Comment by L3LeTrigger-F** [comment]
ok..I solved the problem by setting the learning rate to 1e-2...

**Comment by L3LeTrigger-F** [comment]
new question. When the learning rate is adjusted to 1e-2, the collision rate is 0.99.

**Comment by wzyustc** [comment]
> 新问题。当学习率调整为1e-2时，碰撞率0.99。

您好，请问您的碰撞率是如何下降的呢，我训练时碰撞率一直维持在60%-70%。

**Comment by zhengbw0324** [comment]
@L3LeTrigger-F 
We also found that the loss rose and fell periodically in the experiment, so we tried to use `learning rate schedule`. This will make the learning rate smaller in the final stage of training to achieve better convergence.

**Comment by zhengbw0324** [comment]
@wzyustc 
您好，您的数据的item数量大概是多少呢？如果数量比较大可以尝试：
（1）将4级codebook改为5级，也可以将每级codebook从256个调大。
（2）调整encoder decoder结构，我们实验中发现更深层的encoder有利于降低冲突率。
总的来说，RQ-VAE在建模大量物品时还是存在很多局限性有待解决。

**Comment by Wateau** [comment]
您好，请问一下我训练时的重构损失很高，embedding每个维度均在[-1,1]之间，但是重构损失维持在30多，请问您有遇到这个问题么
